### PR TITLE
Readme ORG

### PR DESCRIPTION
**Pull request by rsm-lisper**
GitHub supports also ORG format. So here it is - README reformatted to org. Should look exactly the same as previous md file.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -37,31 +37,31 @@ For org-static-blog, a blog consists of six parts:
...
-  `org-static-blog-post-postamble` are prepended and appended to every
+- The return values of =org-static-blog-post-preamble= and
+  =org-static-blog-post-postamble= are prepended and appended to every
   blog post. If you want to change the formatting of dates, titles, or
   the tag list, overwrite these functions.
 
 If you want to activate a few convenience key bindings, add
-`(add-to-list 'auto-mode-alist (cons (concat org-static-blog-posts-directory ".*\\.org\\'") 'org-static-blog-mode))`
-to your *init.el*. These key bindings are:
-- `C-c C-f`/`C-c C-b` to open next/previous post.
```
Comment (bastibe) [inline README.org]: this line does not render correctly.

File: README.org [hunk 2]
```diff
@@ -108,31 +99,29 @@ Known Issues
   is available (i.e. the org file is older than the HTML file). If you
```
Comment (bastibe) [inline README.org]: There's a missing link in line 106/97

Comment (rsm-lisper) [reply README.org]: fixed

File: README.org (hunk 1 above)
Comment (rsm-lisper) [reply README.org]: Hmm... I checked source and how it's rendered by GitHub - looks the same for me?...

Comment (bastibe) [reply README.org]: It renders as "C-c C-f=/=C-c C-b" (note the equal signs)

Comment (rsm-lisper) [reply README.org]: Oh, sorry. Fixed :)

**Comment by bastibe** [comment]
Thank you very much!

**Pull request merged**